docs/source/tutorials/cascaded_architecture.rst

```diff
@@ -0,0 +1,154 @@
+model to focus only on the most important part of the image.
+
+.. _Prerequisite:
+
+Prerequisite
+------------
+
+In this tutorial, the spinal cord segmentation model generated from :doc:`../tutorials/one_class_segmentation_2d_unet`
+will be needed. In the log directory of this trained model, a folder named ``seg_sc_t1_t2_t2s_mt`` (if the default value
+of the parameter ``model_name`` was used) contains the packaged model. The path to this folder will be
+needed in this tutorial and will be referenced as the location to the object detection model.
+
+
+Configuration file
+------------------
+
+In this tutorial we will use the configuration file: ``ivadomed/config/config.json``.
+First off, copy this configuration file in your local directory (to avoid modifying the source file):
+
+.. code-block:: bash
+
+   cp <PATH_TO_IVADOMED>/ivadomed/config/config.json .
+
+Then, open it with a text editor. As described in the tutorial :doc:`../tutorials/one_class_segmentation_2d_unet`, make
+sure the ``command`` is set to "train" and ``bids_path`` point to the location of the dataset. Below, we will discuss
+some of the key parameters to use cascaded models.
+
+- ``object_detection_params:object_detection_path``: Location of the object detection model. This parameter corresponds
+  to the prerequisite model path  to the trained model (see :ref:`Prerequisite`). This spinal cord segmentation model
+  will be applied to the images and a bounding box will be created around this mask to crop the image.
+
+  .. code-block:: xml
+
+     "object_detection_path": "<SPINAL_CORD_SEG_LOG_DIRECTORY>/spineGeneric/seg_sc_t1_t2_t2s_mt"
+
+- ``object_detection_params:safety_factor``: Multiplicative factor to apply to each dimension of the bounding box. To
+  ensure all the CSF is included, a safety factor should be applied to the bounding box generated from the spinal cord.
+  A safety factor of 200% on each dimension is applied on the height and width of the image.
+
+  .. code-block:: xml
+
+     "safety_factor": [2, 2, 1]
+
+- ``loader_parameters:target_suffix``: Suffix of the ground truth segmentation. The ground truth are located under the
+  ``DATASET/derivatives/labels`` folder. The suffix for CSF labels in this dataset is ``_csfseg-manual``:
+
+  .. code-block:: xml
+
+     "target_suffix": ["_csfseg-manual"]
+
+- ``loader_parameters:contrast_params``: Contrast(s) of interest. The CSF labels are only available in T2w contrast in
+  the example dataset.
+
+  .. code-block:: xml
+
+     "contrast_params": {
+         "training_validation": ["T2w"],
+         "testing": ["T2w"],
+         "balance": {}
+     }
+
+- ``transformation:CenterCrop:size``: Crop size in pixel. Images will be cropped or padded to fit these dimensions. This
+  allows all the images to have the same size during training. Since the images will be cropped around the spinal cord,
+  the image size can be reduced. The ``preprocessing`` parameter indicates this transformation will only be applied
```

Comment:
Since the image is cropped by the first model, if we don't reduce the size of the image with CenterCrop there will be  large zero padding to meet the size of the CenterCrop (e.i 128x128 will create a large zero padding while 64x64 will have a thinner padding). I added this precision here : 1688087 , but there might be a clearer way to phrase this